### DISCUSSION TITLE
Can't set the correct time

### DISCUSSION THREAD

**Discussion by felipebraz** [Blackymas/NSPanel_HA_Blueprint#2362] [Q&A]
Until yesterday my NSPanels was working fine, but today it stated to get the wrong time. 
At this moment is 13:50 in here and the NSPanels are showing 19:50.
I already tried reviewing the blueprint settings:

![image](https://github.com/user-attachments/assets/1dc8d970-39f7-4c0c-ac26-48f0e628b9d9)

Add the time zone in config file:

`time:
  - id: !extend time_provider
    timezone: "America/Sao_Paulo"
    platform: sntp
    servers:
     - 0.pool.ntp.org
     - 1.pool.ntp.org
     - 2.pool.ntp.org`

and 

`time:
  - id: !extend time_provider
    timezone: "America/Sao_Paulo"`

And reviewed the time on HA server

![image](https://github.com/user-attachments/assets/c9720838-0802-4738-aae8-c29aa10713da)

And everything is correct.

What could be happening?

My blueprint is on v4.3.11 and my HA on 2024.10.x

![image](https://github.com/user-attachments/assets/9f5a9c89-59a1-4791-88bf-385e0637e278)

**Comment by edwardtfn** [comment]
Looks similar to this:
- #2304

**Reply by felipebraz** [reply]
Thank you
